src/components/ResultsDisplay/index.jsx

```diff
@@ -0,0 +1,58 @@
+import RouteCard from "../RouteCard/index.jsx";
+import {useEffect, useState} from "react";
+import PlanRouteButton from "../PlanRouteButton/index.jsx";
+import NoRouteCard from "../NoRouteCard/index.jsx";
+import InvalidInputCard from "../InvalidInputCard/index.jsx";
+import {Link, Routes, Route} from "react-router-dom";
+import ClearRouteButton from "../ClearRouteButton/index.jsx";
+
+function ResultsDisplay({selected, destinationSelected}) {
+
+
+    const [journey, setJourney] = useState([])
+    const [boolean, setBoolean] = useState(false)
+    const [status, setStatus] = useState(0)
+
+    let originCode
+    let destinationCode
+
+    if (selected && destinationSelected) {
+        originCode = selected
+        destinationCode = destinationSelected
+    }
+
+    const fetchJourney = async () => {
+        const response = await fetch(`http://localhost:3000/journeys?origin=${originCode}&destination=${destinationCode}`)
```

Comment:
Also here you want `http://localhost:3000` to be in a single place where you can change it to your production server address. Ideally, it would be in a config.js file somewhere that you can require in wherever it's needed, but don't worry about it for this project.